src/Components/AreaChart.jsx

```diff
@@ -0,0 +1,123 @@
+import {
+  AreaChart,
+  XAxis,
+  YAxis,
+  Area,
+  Tooltip,
+  CartesianGrid,
+  ResponsiveContainer,
+} from "recharts";
+
+import CustomToolTip from "./CustomToolTip";
+import { useSelector } from "react-redux";
+import { LightMode } from "../redux/light-dark/lightDarkTypes";
+import FilterData from "../hooks/FilterData";
+import { Flex, Text } from "@sparrowengg/twigs-react";
+
+const AreaChartComp = () => {
+  const mode = useSelector((store) => store.lightdarkmode.lightDarkMode);
+  const loading = useSelector((store) => store.fakeapidata.loading);
+
+  const { filteredData } = FilterData();
+
+  return (
+    <ResponsiveContainer width={"100%"} height={"100%"}>
+      {loading ? (
+        <Flex
+          css={{
+            marginTop: "25px",
+            height: "320px",
+            width: "100%",
+            border: "1px solid $border",
+            borderRadius: "8px",
+          }}
+          alignItems="center"
+          justifyContent="center"
+        >
+          <Text
+            css={{
+              fontSize: "$2xl",
```

Comment:
@9xVibee instead of having 2xl as fontSize for text can you use heading and use size prop there?